downstream/modules/platform/con-aap-upgrade-planning.adoc

```diff
@@ -7,34 +7,20 @@
 [role="_abstract"]
 Before you begin the upgrade process, review the following considerations to plan and prepare your {PlatformNameShort} deployment:
 
-[discrete]
-== {ControllerNameStart}
-
-* Even if you have a valid license from a previous version, you must provide your credentials or a subscriptions manifest upon upgrading to the latest version of {ControllerName}.
-* If you need to upgrade {RHEL} and {ControllerName}, you must first backup and restore your {ControllerName} data.
-* Clustered upgrades require special attention to instance and instance groups before upgrading.
+* See link:{URLPlanningGuide}/platform-system-requirements[System requirements] in the {TitlePlanningGuide} guide to ensure you have the topologies that fit your use case. 
++
+[NOTE]
+====
+2.4 to 2.5 upgrades now include link:{URLPlanningGuide}/ref-aap-components#con-about-platform-gateway_planning[Platform gateway]. Ensure you review the 2.5 link:{URLPlanningGuide}/ref-network-ports-protocols_planning[Network ports and protocols] for architectural changes.
+====
++
+* Ensure you have a valid subscription from a previous version. You must provide your credentials or a link:https://access.redhat.com/articles/5807761[subscriptions manifest] upon upgrading to the latest version of {PlatformNameShort}.
+* Clustered upgrades require special attention to instance and instance groups before upgrading. Ensure you capture your inventory or instance group details before upgrading.
+* Upgrading 2.4 {EDAName} is not supported.[ADD FURTHER TEXT HERE]
```

Comment:
@hherbly A reminder to add the additional text.

Reply:
I definitely will. This is a spot where Satoe commented.